js/helper.js

```diff
@@ -348,6 +391,84 @@ function getProjectFlagsFromOptions({ projects = [] } = {}) {
   return projects.reduce((flags, project) => flags.concat('-p', project), []);
 }
 
+/**
+ * Determines an appropriate success message based on the command.
+ *
+ * @param {string[]} args Command line arguments passed to sentry-cli
+ * @returns {string|null} Success message to display or null if no message needed
+ */
+function determineSuccessMessage(args) {
+  if (!args || args.length === 0) {
+    return null;
+  }
+
+  const command = args[0];
+
+  // Only show success messages for high-impact operations customers care about
+  switch (command) {
+    case 'releases':
+      if (args[1] === 'new' && args[2]) {
+        return `✓ Release ${args[2]} created`;
+      } else if (args[1] === 'finalize' && args[2]) {
+        return `✓ Release ${args[2]} finalized`;
+      } else if (args[1] === 'files' && args[3] === 'upload-sourcemaps' && args[2]) {
+        return `✓ Source maps uploaded to release ${args[2]}`;
+      }
+      break;
+
+    case 'sourcemaps':
+      if (args[1] === 'upload') {
+        return `✓ Source maps uploaded`;
+      } else if (args[1] === 'inject') {
+        return `✓ Source maps injected`;
+      }
+      break;
+
+    case 'debug-files':
+      if (args[1] === 'upload') {
+        return `✓ Debug files uploaded`;
+      }
+      break;
+
+    case 'upload-proguard':
+      return `✓ ProGuard mappings uploaded`;
+
+    case 'upload-dif':
+      return `✓ Debug information files uploaded`;
+
+    case 'upload-dsym':
+      return `✓ dSYM files uploaded`;
```

Comment:
These commands are both deprecated aliases of `debug-files upload`, so output should be the same

```suggestion
    case 'upload-dif':
    case 'upload-dsym':
      return `✓ Debug files uploaded`;
```